frontend/src/app/utils/ApiUrl.ts

```diff
@@ -2,7 +2,7 @@ export enum API_URL {
     LOGIN = '/auth/sign-in',
     REGISTER = '/auth/register',
     USER_INFO = '/users/info',
-    DATASETS = '/scans/datasets',
+    DATASETS = '/datasets/',
```

Comment:
Hi @piotrwalkusz1 please delete this slash. As far as I know it is not correct and it shouldn't be there.

Reply:
done